RetroBar/PropertiesWindow.xaml

```diff
@@ -298,6 +297,72 @@
                     </GroupBox>
                 </StackPanel>
             </TabItem>
+            <TabItem Header="{DynamicResource clock_tab}" >
+                <StackPanel Orientation="Vertical"
+                            Margin="10">
+                    <DockPanel>
+                        <GroupBox Header="{DynamicResource clock_appearance}"
+                                  Margin="0,0,0,10">
+                            <StackPanel Orientation="Vertical">
+                                <ContentControl Template="{StaticResource NotificationAreaPreview}"
```

Comment:
The bug with the preview seems to happen because the clock control has no children at the time the `Loaded` event fires, so the ConverterCulture doesn't get set. I'm not sure why it is happening in this specific case.

IMO this should be fixed before the changes can be merged. There is an alternative, we could remove the extra preview, but that would make the Clock tab a bit sparse. Together with replacing the explanatory string with a link, the clock format section could fit on the Advanced tab instead.